k8s-multus/scripts/node01.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+# Wait for the network to really came up
+while [[ `cat /proc/sys/net/ipv4/ip_forward` -eq 0 ]]
+do
+ sleep 2
+done
+
+while [[ ! -f /proc/sys/net/bridge/bridge-nf-call-iptables ]]
+do
+ sleep 2
```

Comment:
@slintes looks like this may be related to the issues you saw?

Reply:
Ok, then it might be better to wait for docker in general?

Reply:
Done